ups/deploy_testing_distrib.table

```diff
@@ -0,0 +1,4 @@
+setupRequired(meas_deblender)
+setupRequired(obs_test)
+setupOptional(obs_cfht)
+setupOptional(obs_subru)
```

Comment:
Yeah I saw that. You have a better recommendation? I think the MVP should at least use the compiler and not be a external dependency.